autoscaler/controllers/datacollection/daemonset.go

```diff
@@ -37,10 +38,27 @@ var (
 	}
 )
 
+func getOdigletDaemonsetPodSpec(ctx context.Context, c client.Client, namespace string) (*corev1.PodSpec, error) {
+	odigletDaemonset := &appsv1.DaemonSet{}
+
+	if err := c.Get(ctx, client.ObjectKey{Namespace: namespace, Name: resources.OdigletDaemonSetName}, odigletDaemonset); err != nil {
```

Comment:
We are polling the entire cli module as a dependency just to get the `resources.OdigletDaemonSetName` string which is not likely to ever change.

I wonder if we can just define this const here and skip adding a new dependency. WDYT?

Reply:
done

Reply:
a better approach would be create 1 module that will be shared over all the rest of modules in order to keep things standard I guess?

Reply:
> a better approach would be create 1 module that will be shared over all the rest of modules in order to keep things standard I guess?

Yes I was thinking the same thing. Perhaps we will do it one day and cleanup a bit